frontend/src/views/ShootDetails.vue

```diff
@@ -99,12 +91,7 @@ export default {
     shootItem () {
       return get(this, 'value', {})
     },
-    isLoggingFeatureGateEnabled () {
-      const userCredentialsAvailable = !!this.info.logging_username && !!this.info.logging_password
-      const adminCredentialsAvailable = !!this.seedInfo.logging_username && !!this.seedInfo.logging_password
-      return userCredentialsAvailable || adminCredentialsAvailable
-    },
-    tickets () {
+    journals () {
```

Comment:
this seems to be a merge issue
```suggestion
    tickets () {
```

Reply:
@petersutter

Reply:
Yes, it is maybe a issue from the rebasing. Gonna fix it :)

Reply:
correct, @grolu